### PR TITLE
Using themes

### PR DESCRIPTION
**Pull request by apreshill**
This is a fairly opinionated section on using Hugo themes, intended for the "Get started" chapter. Depends on #618 and #619 coming before it.

### CODE REVIEW

File: docs/01-introduction.Rmd [hunk 1]
```diff
@@ -341,16 +341,23 @@ If you are working on an R Markdown post, but do not want **blogdown** to compil
...
+1. Configuration files. This may be a single `config.yaml` or `config.toml` file, or some themes use a configuration directory also located in `config/_default/`. Read more about these files in Chapter \@ref(configuration).
```
Comment (cderv) [inline docs/01-introduction.Rmd:346]: Curious: are themes with config directory working with **blogdown** ? 
I am not sure this is yet supported: https://github.com/rstudio/blogdown/issues/611

If it is not working yet, maybe we should warn to not choose a them with a config dir 🤷‍♂️

**Review by cderv — COMMENTED** [review]
That looks good! Thanks!

File: docs/01-introduction.Rmd (hunk 1 above)
Comment (apreshill) [reply docs/01-introduction.Rmd:346]: Wowchemy has this structure, and the config.toml gets copied to the root. But other config options remain in the directory, see: https://alison.rbind.io/post/new-year-new-blogdown/#configure-your-site

Comment (cderv) [reply docs/01-introduction.Rmd:346]: Thanks for the clarification.

Comment (apreshill) [reply docs/01-introduction.Rmd:346]: Let me test with `install_theme()` though, I could encourage use of the `update_config=FALSE` arg there.

**Pull request merged**